src/westpa/core/_rc.py

```diff
@@ -322,6 +322,11 @@ def new_sim_manager(self):
             from .binning.mab_manager import MABSimManager
 
             sim_manager = MABSimManager(rc=self)
+        elif binmapname == 'BinlessMapper':
```

Comment:
Was `binmapname` defined before?

Reply:
I just had a quick look at the git history. It seems like when I was resolving the conflict for `_rc.py`, I've brought back some old lines that were deleted in the latest commit. I'll push a new commit to fix that!